### ISSUE TITLE
Key create_train_op/loss/replica not found in checkpoint

### ISSUE THREAD

**Issue by HuihuiChyan** [THUNLP-MT/L2Copy4APE#3]
While doing the second step of training(namely interactive+CopyNet+Predictor+Joint Training), an error occur like this:

```
NotFoundError (see above for traceback): Restoring from checkpoint failed. This is most likely due to a Variable name or other graph key that is missing from the checkpoint. Please ensure that you have not altered the graph expected based on the checkpoint. Original error:

Key create_train_op/loss/replica not found in checkpoint
         [[{{node save/RestoreV2}} = RestoreV2[dtypes=[DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, ..., DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT], _device="/job:localhost/replica:0/task:0/device:CPU:0"](_arg_save/Const_0_0, save/RestoreV2/tensor_names, save/RestoreV2/shape_and_slices)]]
```
Do you have any idea why this is happening?
Sorry to ask this stupid question, but I am really not familiar with python2 and tensorflow1.11. I would appreciate it if you can give me any suggestions.

**Comment by xc-kiwiberry** [comment]
Are you using tensorflow 1.11？Maybe it is caused by the mismatched version of tensorflow.
Or you may need to check the checkpoint which should be pre-trained in step-1.
